.github/workflows/dependabot.yml

```diff
@@ -0,0 +1,55 @@
+name: "Dependabot Workflow"
+on: [pull_request_target] # Dependabot forces this for secrets access
+
+jobs:
+  publish-tauri:
+    if: ${{ github.actor == 'dependabot[bot]' }}
```

Comment:
ℹ️ This is the only different thing from the PR workflow besides the target. Only runs if Dependabot is the actor.